### PR TITLE
add ASL property to user

### PR DESCRIPTION
**Pull request by HoseinHaqiqian**
This only adds isASL property to user model

### CODE REVIEW

File: lib/src/models/user.dart [hunk 1]
```diff
@@ -54,24 +56,25 @@ class User {
...
+          String? lastName,
+          List<String>? linkedAccounts,
+          String? phoneNumber,
+          bool? tosAccepted,
+          List<Profile>? profiles,
+          String? referralLink,
+          int? aiDailyMessageLimit,
+          AiVerbosity? aiVerbosity,
+          AiLanguageLevel? aiLanguageLevel,
+          bool? isASL}) =>
```
Comment (ming-chu) [inline lib/src/models/user.dart:77]:
```suggestion
          bool? isASL,
       }) =>
```

Comment (HoseinHaqiqian) [reply lib/src/models/user.dart]: Eagle's eyes. resolved.

**Review by bwang-aira — APPROVED** [review]

**Review by ming-chu — APPROVED** [review]

**Review by ming-chu — DISMISSED** [review]

**Review by evgeniya-aira — COMMENTED** [review]
Please update the `CHANGELOG.md` file and upgrade the version.

**Comment by HoseinHaqiqian** [comment]
> Please update the `CHANGELOG.md` file and upgrade the version before merge

I believed we let GitHub do versioning management. anyway, its done. please approve pull request if it looks good to you

**Review by ming-chu — APPROVED** [review]

**Review by kasemj — APPROVED** [review]
LGTM

**Pull request merged**